supabase/tables/base.ts

```diff
@@ -0,0 +1,12 @@
+import { SupabaseClient } from "@supabase/supabase-js";
+import { Logs } from "ubiquibot-logger";
+
+export class SuperBase {
+  protected supabase: SupabaseClient;
+  public logger: Logs;
+
+  constructor(supabase: SupabaseClient) {
+    this.supabase = supabase;
+    this.logger = {} as Logs;
```

Comment:
Why not `new Logger()`?

Reply:
I avoided `new Logger()` to keep the constructor simple and avoid unnecessary configuration which are not optional. Instead, I initialized logger as an empty object for simplicity.

Reply:
So are the inheritors responsible to initialize the logger? What happens if a class does not initialize it?

Reply:
You're right. Inheritors need to initialize the logger, or it will stay as an empty object and could cause issues.